_i18n/es/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,166 @@
 
 `monero-wallet-cli.exe --generate-from-device MoneroWallet --subaddress-lookahead 3:200` (Win 7 + 8)
 
 `.\monero-wallet-cli.exe --generate-from-device MoneroWallet --subaddress-lookahead 3:200` (Win 10)
 
-4. The CLI will, after executing aforementioned command, prompt your for a password. Make sure to set a strong password and confirm it thereafter.
+4. La consola te preguntará, después de ejecutar el comando anterior, por una contraseña. Asegúrate de establecer una contraseña fuerte y de confirmarla después de ello.
 
-5. The Ledger will ask whether you want to export the private view key or not. First and foremost, your funds cannot be compromised with merely the private view key. Exporting the private view key enables the client (on the computer - Monero v0.12.2.0) to scan blocks looking for transactions that belong to your wallet / address. If this option is not utilized, the device (Ledger) will scan blocks, which will be significantly slower. There is, however, one caveat. That is, if your system gets compromised, the adversary will potentially be able to compromise your private view key as well, which is detrimental to privacy. This is virtually impossible when the private view key is not exported.
+5. Ledger preguntará si deseas exportar la clave privada o no. Primero y ante todo, tus fondos no pueden ser comprometidos con simplemente tu clave privada de visualización. Exportar la clave privada de visualización permite al cliente (en el ordenador - Monero v0.12.2.0) escanear bloques en busca de transacciones pertenecientes a tu monedero / dirección. Si esta opción no es utilizada, el dispositivo (Ledger) escaneará bloques, lo que será significativamente más lento. Sin embargo, hay una advertencia. Esta es, si tu sistema se ve comprometido, el adversario será potencialmente capaz de también comprometer tu clave privada, lo que es perjudicial para tu privacidad. Esto es virtualmente posible cuando la clave privada no es exportada.
 
-6. You may have to hit confirm twice before it proceeds.
+6. Puedes tener que hacer clic en confirmar dos veces antes de proceder.
 
-7. Your Ledger Monero wallet will now be generated. Note that this may take up to 5-10 minutes. Furthermore, there will be no immediate feedback in the CLI nor on the Ledger.
+7. Tu monedero Ledger Monero será ahora generada. Ten en cuenta que esto puede tomar algunos 5-10 minutos. Además, no habrá realimentación inmediata en la consola de comandos o en la Ledger.
 
-8. `monero-wallet-cli` will start refreshing. Wait until it has fully refreshed.
+8. `monero-wallet-cli` comenzará a refrescarse. Espera a que se haya refrescado totalmente.
 
-Congratulations, you can now use your Ledger Monero wallet in conjunction with the CLI.
+Felicidades, ahora puedes utilizar tu monedero Ledger Monero en conjunto con la consola de comandos.
 
 ### 2. Mac OS X
-We first have to ensure that we're sufficiently prepared. This entails the following:
```

Comment:
Paro acá. Encontre nuevamente los problemas de las lineas en la traducción. 
A partir de la linea 59 en rojo:
1. This guide assumes you have already initialized your Ledger wallet and thus generated a 24 word mnemonic seed.
Le sigue la traduccion 58 en verde:
Primero necesitamos asegurarnos de estar suficientemente preparados. Esto implica lo siguiente:

Aqui podría volver a tener conflictos cuando se `merge` las traducciones.

Reply:
Le agregué un espacio porque no lo tenía, pero el número 1 y 3 sí lo tienen.

There are a lot of similar cases, so I'm not sure if it's because something of it's just a little mistake. In other files there are double line jumps


like this, but in other paragraphs the jumps are

simple, like this one. So I can't really say.

Reply:
I'll build the site again now that it works to see the changes myself.

Reply:
Pero si viste que en el archivo las traducciones se cruzaron otra vez como se dio en el mismo archivo la semana pasada? Pare la revisión precisamente por eso porque no podría especificar con claridad cual traducción es la que se hace referencia.

Reply:
Si claro, pero no es batallar con el archivo. Me conscierne es ver como se cruzan y los posibles errores que emerjan al tratar de merge y despues toque volver a revisar. Quedo ya atento. :) graciassss!